docs/data/data-grid/server-side-data/lazy-loading.md

```diff
@@ -2,14 +2,107 @@
 title: React Server-side lazy loading
 ---
 
-# Data Grid - Server-side lazy loading [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')🚧
+# Data Grid - Server-side lazy loading [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
 
 <p class="description">Row lazy-loading with server-side data source.</p>
 
+Lazy Loading changes the way pagination works by removing page controls and loading data dynamically (in a single list) as the user scrolls through the grid.
+
+It is enabled by adding `lazyLoading` prop in combination with `unstable_dataSource` prop.
+
+Initially, the first page data is fetched and displayed in the grid. What triggers the loading of next page data depends on the value of the total row count.
+
+If the total row count is known, the grid gets filled with skeleton rows and fetches more data if one of the skeleton rows falls into the rendering context.
```

Comment:
IIRC, we prefer to use full product names rather than the shortened ones. 
CC @samuelsycamore 
```suggestion
If the total row count is known, the Data Grid gets filled with skeleton rows and fetches more data if one of the skeleton rows falls into the rendering context.
This loading strategy is often referred to as [**viewport loading**](#viewport-loading).
```

Reply:
+1 - you could also just write "Grid" if it's too repetitive to say "Data Grid" over and over, but either way it's preferable to capitalize it to make it clear that it's the MUI X product specifically.